frontend/src/components/VSkeleton/VGridSkeleton.vue

```diff
@@ -63,14 +63,13 @@ export default defineComponent({
 
 <style scoped>
 .square {
-  aspect-ratio: 1 / 1;
+  @apply aspect-square;
 }
 
 .masonry {
-  column-count: 2;
-  column-gap: 1rem;
+  @apply columns-2 gap-x-4 md:columns-3 lg:columns-5;
 
-  @screen md {
+  .masonry {
```

Comment:
```suggestion
```

We already set the column count to 3 in the `.masonry` declaration in line 70 (`md:columns-3`), so lines 72 and 73 should be deleted.